### PR TITLE
Add github action to docs

### LINKED ISSUE #154: Add GHA to docs
Add microsoft/pybryt-action to docs.

### PR DESCRIPTION
**Pull request by chrispyles**
Closes #154

### CODE REVIEW

**Review by leestott — DISMISSED** [review]

**Comment by leestott** [comment]
awaiting changes to github action before approval/merge is completed.

**Comment by chrispyles** [comment]
@leestott this is ready for review/merge

**Review by leestott — APPROVED** [review]

**Pull request merged**
